### ISSUE TITLE
[Homepage and search pages] Add alt text to magnifying glass icon

### ISSUE THREAD

**Issue by sboots** [ytgov/ckanext-yukondesign#110]
**Estimated hours: 1**

The magnifying glass icon SVG image used in the search boxes should have alt text (which could just be "magnifying glass"). This should be updated on both the homepage search boxes and the search pages for each schema type.

![Image](https://github.com/user-attachments/assets/062ec483-ce18-489e-ab3a-94b5cb6a7268)
